### ISSUE TITLE
Determine if we need these handlers

### ISSUE THREAD

**Issue by tfplenum** [tfplenum/rock#12]
<a href="https://github.com/grantcurell"><img src="https://avatars2.githubusercontent.com/u/1595564?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [grantcurell](https://github.com/grantcurell)**
_Monday Jan 01, 2018 at 16:37 GMT_
_Originally opened as https://github.com/grantcurell/rock/issues/33_

----

The original rock code has these two handlers, but they were unused. Need to determine before we button everything up if they are needed. They were in the original deploy rock.

```
  #######################################################
  #####################  Handlers  ######################
  #######################################################
# NOT CURRENTLY IN USE
#  handlers:
#    - name: force sync time
#      command: >
#        chronyc -a 'burst 3/4'; sleep 5; chronyc -a makestep
#
#    - name: configure monitor interfaces
#      shell: >
#        for intf in {{ rock_monifs | join(' ') }}; do
#          /sbin/ifup ${intf};
#        done
#
```

**Comment by GonzoVC** [comment]
TL;DR:
-----
It appears the code was transferred to the "common" and "sensor-common" roles. So the code is present and in use, but it is all still a work in progress.

Closing Issue.

Install Test
----
I ran the installer and ensured the services it provided were still included.
The time sync portion of the is done by:
../playbooks/roles/common/tasks/configure_ntp.yml
../playbooks/roles/common/tasks/install_chrony.yml
../playbooks/roles/common/tasks/install_common_packages.yml

Time before install:
```
[root@localhost ~]# timedatectl
      Local time: Thu 2018-01-11 12:52:01 CST
  Universal time: Thu 2018-01-11 18:52:01 UTC
        RTC time: Thu 2018-01-11 18:52:01
       Time zone: America/Chicago (CST, -0600)
     NTP enabled: n/a
NTP synchronized: no
 RTC in local TZ: no
      DST active: no
 Last DST change: DST ended at
                  Sun 2017-11-05 01:59:59 CDT
                  Sun 2017-11-05 01:00:00 CST
 Next DST change: DST begins (the clock jumps one hour forward) at
                  Sun 2018-03-11 01:59:59 CST
                  Sun 2018-03-11 03:00:00 CDT

[root@localhost ~]# date
Thu Jan 11 12:52:04 CST 2018
```

Chrony/NTP actions completed during install:
```
TASK [common : include_tasks] ****************************************************************************************
included: /opt/rock/playbooks/roles/common/tasks/configure_ntp.yml for simplerockbuild.simplerock.lan

TASK [common : Enable and start chrony] ******************************************************************************
changed: [simplerockbuild.simplerock.lan]

TASK [common : Set system timezone] **********************************************************************************
changed: [simplerockbuild.simplerock.lan]

TASK [common : Check if RTC set to UTC] ******************************************************************************
ok: [simplerockbuild.simplerock.lan]

TASK [common : Set system hardware clock to UTC] *********************************************************************
skipping: [simplerockbuild.simplerock.lan]
.
.
.
TASK [common : Install Chrony package] *******************************************************************************
ok: [simplerockbuild.simplerock.lan]
```

Time after install:
```
[root@localhost bin]# timedatectl
      Local time: Thu 2018-01-11 19:01:53 UTC
  Universal time: Thu 2018-01-11 19:01:53 UTC
        RTC time: Thu 2018-01-11 19:01:53
       Time zone: UTC (UTC, +0000)
     NTP enabled: yes
NTP synchronized: yes
 RTC in local TZ: no
      DST active: n/a

[root@localhost bin]# date
Thu Jan 11 19:00:53 UTC 2018
```

The configurations for monitor interfaces is now located in:
../playbooks/roles/sensor-common/handlers/main.yml
../playbooks/roles/sensor-common/tasks/main.yml
../playbooks/roles/sensor-common/templates/*

Interfaces configurationn during install:
```

RUNNING HANDLER [sensor-common : configure monitor interfaces] *******************************************************
```
